group_vars/all

```diff
@@ -102,33 +103,34 @@ tarsnapper:
     schedule: hourly
     run_on: trusted
 
-gitannex:
-    stop_on_untrusted: True
+# no arm pkg, could use git-annex-git?
+#gitannex:
+#    stop_on_untrusted: True
 
 bitlbee:
-    run_on: trusted
-    torify: True
+    run_on: none
+    torify: False
 
-bootloader: grub
+bootloader: arm
 
 postgresql:
-    enable: True
+    enable: False
 
 fingerprint: False
 
 tor:
-    run_on: trusted
+    run_on: none
 
 localtime:
-    run_on: trusted
+    run_on: all
 
 nameservers:
     - 8.8.8.8       # Google
     - 209.244.0.3   # Level 3
     - 74.82.42.42   # Hurricane Electric
 
 bluetooth:
```

Comment:
Generally speaking, my r-pi is stationary and only connected to one network. I suppose I could leave things like `localtime` to still run on `trusted` and define my single trusted network. Let me go that route to make it a cleaner diff against master.

For Bluetooth, I was going to experiment with it and get some bt speakers working, and should hopefully get that section of group_vars/all back to matching with master.